tools/redefinitions.mk

```diff
@@ -0,0 +1,15 @@
+saxpath = "$(HOME)/java/saxon-9-7/saxon9he.jar"
```

Comment:
Maybe you should just warn if this isn't set in the environment.  The circle build sets it and anyone using this can as well.